config/configuration.json

```diff
@@ -723,6 +723,7 @@
     }
   },
   "RoundaboutMissingTagCheck": {
+    "enabled": false,
```

Comment:
Unless there's a glaring issue with performance, I'm okay with leaving all checks enabled in this repo

Reply:
The check has high FP based on recent feedback from analysis team so was considering disabling it here. There isn't any known issue with performance.